src/SSW.SophieBot/SSWSophieBot.UtcConvertAction/SSWSophieBot.UtcConvertAction.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Does this need to be a class lib? try net standard 2.0 or is this setup defined by Microsoft?

Reply:
Changed to net standard 2.1